insights/client/apps/malware_detection/__init__.py

```diff
@@ -863,7 +875,13 @@ def scan_processes(self):
                 logger.debug("Unable to scan process %s: %s", scan_pid, cpe.output.strip())
                 continue
 
-            self.parse_scan_output(output)
+            try:
+                self.parse_scan_output(output)
+            except Exception as e:
+                self.potential_matches += 1
+                logger.error("Rule match(es) potentially found in process %s but problems encountered parsing the results: %s.  Skipping ...",
+                             scan_pid, str(e))
+                traceback.print_exc()
```

Comment:
@mhuth do you really want to print this to STDOUT?

Reply:
@bfahr I figured it could be one less troubleshooting step that needed to be done if something completely unexpected went wrong.  But I can remove it if its not very insights-client-y

Reply:
@mhuth I just thought it might be better to expose it in a log message so it would be captured with other run information.

Reply:
Ah ok, good point Bob.  I'll update the PR so it logs the tracebacks

Reply:
@bfahr I'm using logger.exception(<msg>) now to print a traceback to stderr, so it gets displayed on the command line and in the logs.  Thanks for picking this up.